vncviewer/org.tigervnc.vncviewer.metainfo.xml.in

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  (c) 2021-2022 Joachim Falk <joachim.falk@gmx.de>
+
+  Copying and distribution of this file, with or without modification, are
+  permitted in any medium without royalty provided the copyright notice and this
+  notice are preserved. This file is offered as-is, without any warranty.
+-->
+<component type="desktop-application">
+  <id>org.tigervnc.vncviewer</id>
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  <name>TigerVNC Viewer</name>
+  <summary>Remote desktop access using the VNC desktop sharing system</summary>
+  <content_rating type="oars-1.1"/>
+  <releases>
```

Comment:
I'm not looking forward to maintaining this as well, so I prefer if we remove it. On my Fedora machine it's about half that include this, so it doesn't seem to be considered mandatory. :)

Reply:
To what are you exactly referring to? The <content_rating type="oars-1.1"/>?

Reply:
The release information. I would prefer if this file can be kept fairly static.

Reply:
Release information has been removed.